client/ayon_shotgrid/addon.py

```diff
@@ -80,6 +80,23 @@ def is_local_storage_enabled(self):
     def get_local_storage_key(self):
         return self._local_storage_key or None
 
+    def get_credentials(self):
+        from .lib import credentials
+
+        if self._client_login_type == "env":
+            user_login = (
+                os.getenv("AYON_SG_USERNAME")
+                # TODO: Remove USER env variable in future once ayon-core deadline
+                # passing of AYON_SG_USERNAME is solved
+                or os.getenv("USER")
+            )
```

Comment:
According to @fabiaserra [here](https://github.com/ynput/ayon-shotgrid/pull/118/files#r1692014726) we can remove the fallback.
```suggestion
            user_login = os.getenv("AYON_SG_USERNAME")
```

We'd have to need to update the TESTING NOTES to include:

1. Test with all different types of login options - they must work locally and on farm publishes.
    - [ ] env local
    - [ ] env farm
    - [ ] tray api key local
    - [ ] tray api key farm

(not sure if those are the actual login options, so consider it just an example of what to add)

Reply:
@jakubjezek001  could confirm if the mentioned issue in the todo was solved ?

Reply:
Yeah it is perhaps possible, since we are already passing `AYON_SG_USERNAME` in deadline.

Reply:
is that how the tray login is passing the username as well? I wonder why Steve on Discord mentioned his publish didn't work as the `AYON_SG_USERNAME` should have been set in the Deadline task and also collected?